recipe/meta.yaml

```diff
@@ -14,6 +14,7 @@ source:
 build:
   number: 0
   script: python setup.py install --single-version-externally-managed --record record.txt
+  noarch: python
```

Comment:
Normally we bump the build number when making this change to avoid a `conda` warning. It's not crucial, but something to keep in mind for the future.